js/glpi_dialog.js

```diff
@@ -303,13 +303,10 @@ var glpi_confirm = function({
 
 
 /**
- * Close and remove from dom all opened glpi dialog
+ * Remove from dom all opened glpi dialog
  */
 var glpi_close_all_dialogs = function() {
-   document.querySelectorAll('.modal').forEach(function(modalElem) {
-      const myModal = new bootstrap.Modal(modalElem);
-      myModal.hide();
-   });
+   $('.modal.show, .modal-backdrop.show').remove();
```

Comment:
```suggestion
   $('.modal).modal('hide');
```
This seems to work as an alternative. It isn't destructive so modals that are built once can still be re-shown and it will still trigger the `hide.bs.modal` event.

Reply:
indeed, i'll check again #10286 because i originaly found issue with modal before this pr

Reply:
strange, i cannot trigger the original issue